docs/roadmap.md

```diff
@@ -0,0 +1,169 @@
+# Roadmap
+
+* [**Simple installation**](#simple-installation): importing a project should be
+  simple and require as few steps as possible.
+* [**Correct diagnostics**](#correct-diagnostics): the editor should never show
+  spurious red squiggles, even at the cost of higher latency.
+* [**Robust navigation**](#robust-navigation): goto definition should work
+  everywhere, even for Java dependencies.
+* [**Fast completions**](#fast-completions): completions suggestions should
+  respond in ~200ms, even at the cost of occasionally wrong results.
+* [**Low CPU and memory usage**](#low-cpu-and-memory-usage): indexing should be
+  lightweight and run in the background.
```

Comment:
What does "lightweight" mean here? Much like "principled", this word is very often abused, to the degree that I'm not really sure what it means anymore.

Reply:
I disagree "lightweight" has no meaning. Nevertheless, rephrased to

> Indexing should run in the background and not get in your way from coding.